airflow/providers/apache/hive/hooks/hive.py

```diff
@@ -981,15 +981,13 @@ def to_csv(
 
         self.log.info("Done. Loaded a total of %s rows.", i)
 
-    def get_records(
-        self, hql: str, schema: str = 'default', hive_conf: Optional[Dict[Any, Any]] = None
-    ) -> Any:
+    def get_records(self, sql: Union[str, List[str]], parameters: Optional[Any] = None, **kwargs) -> Any:
```

Comment:
```suggestion
    def get_records(
        self,
        sql: Union[str, List[str]],
        parameters: Optional[Any] = None,
        *,
        schema: str = "default",
        **kwargs,
    ) -> Any:
```

We can keep `default` here (instead of needing to dig it out from `kwargs`)

Reply:
Right.

Reply:
Actually:

```
"get_records" incompatible with supertype "DbApiHook"  [override]
        def get_records(
        ^
airflow/providers/apache/hive/hooks/hive.py:984: note:      Superclass:
airflow/providers/apache/hive/hooks/hive.py:984: note:          def get_records(self, sql: Union[str, List[str]], parameters: Optional[Any] = ..., **kwargs: Dict[Any, Any]) -> Any
airflow/providers/apache/hive/hooks/hive.py:984: note:      Subclass:
airflow/providers/apache/hive/hooks/hive.py:984: note:          def get_records(self, sql: Union[str, List[str]], parameters: Optional[Any] = ..., *, schema: str = ..., **kwargs: Any) -> Any
```

I'd like to avoid overloads here so possibly the scheme kwars was a bit better